internal/appsec/config.go

```diff
@@ -71,7 +71,13 @@ func readWAFTimeoutConfig() (timeout time.Duration) {
 	}
 	parsed, err := time.ParseDuration(value)
 	if err != nil {
-		logEnvVarParsingError(wafTimeoutEnvVar, value, err, timeout)
+		// Retry by adding the microsecond suffix us which is the default suffix
+		retry, err2 := time.ParseDuration(value + "us")
```

Comment:
This is not a hot path and perf shouldn't matter here so I won't block on this:
I'd be of the opinion to check the value last char and add "us" by default if the last char is not a letter, which simplifies down to one call to time.ParseDuration and also simplifies error handling.